frontend/src/Landing/Landing.jsx

```diff
@@ -3,8 +3,53 @@ import './Landing.css'
 export default function Landing() {
   return (
     <>
-      <nav></nav>
+      <nav className='navbar'>
```

Comment:
do we want to pull out the navigation bar and create a separate component for it - assuming the nav bar will appear on multiple different pages? 

with a component, we could probably also customize which tabs appear (i.e. patients and physicians will see different tabs or the logged out page might not have "My Profile")

Reply:
Never thought of that. Working on it!